src/aktualizr_get/main.cc

```diff
@@ -70,7 +73,9 @@ int main(int argc, char *argv[]) {
     if (commandline_map.count("header") == 1) {
       headers = commandline_map["header"].as<std::vector<std::string>>();
     }
-    std::string body = aktualizrGet(config, commandline_map["url"].as<std::string>(), headers);
+    std::string body = aktualizrGet(config, commandline_map["url"].as<std::string>(), headers,
+				    commandline_map.count("storage") && commandline_map["storage"].as<std::string>() == "Uptane" ?
```

Comment:
not critical. maybe it makes sense to add 'if' statement for determining of the storage value as it's done for `headers`, it just makes debugging and reading slightly easier, IMO.

Reply:
ok fixed